source/sass/patterns/atoms/doi.scss

```diff
@@ -0,0 +1,8 @@
+@import "../../mixins/typography";
+@import "../../settings/color";
+@import "../../settings/font";
+@import "../../vendor/sass-mq/mq";
+
```

Comment:
This file is no longer needed.

Reply:
We will still need to have to change the reference_doi so it takes in the doi mixin

Reply:
Yes please.